src/utils/tokenizers.py

```diff
@@ -0,0 +1,59 @@
+"""
+Tokenizer class
+To add a tokenizer, add to the below inherting from
+main Tokenizer class.
+
+"""
+import os
+from nltk.tokenize.moses import MosesTokenizer as NLTKMosesTokenizer, MosesDetokenizer
+
+class Tokenizer(object):
+
+    def tokenize(self, sentence):
+        raise NotImplementedError
+
+class OpenAIBPETokenizer(Tokenizer):
```

Comment:
Not necessary now, but it might be nice to have `detokenize()` implementations on the BPE tokenizers as well. Maybe add a TODO?